app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt

```diff
@@ -603,6 +649,8 @@ class GleanMetricsService(private val context: Context) : MetricsService {
             showSearchShortcuts.set(context.settings().shouldShowSearchShortcuts.toStringList())
             openLinksInAPrivateTab.set(context.settings().openLinksInAPrivateTab.toStringList())
             searchSuggestionsPrivate.set(context.settings().shouldShowSearchSuggestionsInPrivate.toStringList())
+            showVoiceSearch.set(context.settings().shouldShowVoiceSearch.toStringList())
```

Comment:
Are these new additions to the pref-toggle-event or prefs-metrics ping? If so please document. Or was it one of the missing pings?

Reply:
Nope! Part of the initial telemetry I forgot.